consensus/polybft/mocks.go

```diff
@@ -0,0 +1,118 @@
+package polybft
```

Comment:
You can name this file `mocks_test.go` instead of just `mocks`, so it won't get compiled into the final binary 

cc @vcastellm